### ISSUE TITLE
The last paragraph of 7.3.2.2 om EPUB A11Y 1.1

### ISSUE THREAD

**Issue by murata2makoto** [mattgarrish/epub-revision#1897]
>the entirety of each EPUB Content Document MUST achieve the conformance level and every Content Document in the EPUB Publication MUST meet the stated conformance level

I think that this statement repeats the same requirement twice.

**Comment by mattgarrish** [comment]
The first "must" applies to individual content documents -- you can't claim partial conformance for any given document, not fully check a document, etc.

The second "must" applies to the set of content documents in the publication -- you can't have only a few documents that pass and claim conformance; they all have to pass.

I think we can simplify this statement, though, as I'm not sure the "must" statements are completely necessary given this is only a clarification:

> The "Full Pages" requirement [WCAG2] -- that parts of a page cannot be excluded when making a conformance claim -- applies to every Content Document in the EPUB Publication (i.e., they must all conform in full to the conformance level claimed).
